core/src/test/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProviderSpec.scala

```diff
@@ -496,15 +611,15 @@ class CompactEntityProviderSpec extends TestDriverComponentWithFlatSpecAndMatche
         )
       )
     )
-    val entity = Entity("name", "type", attributes)
+    val entity = Entity("type", "name", attributes)
     val provider = providerWithMocks(mock[slickDataSource.dataAccess.compactEntityQuery.type])
 
-    val actual: Map[AttributeName, Seq[AttributeEntityReference]] = provider.findAllReferences(entity)
+    val actual: Map[AttributeEntityReference, Seq[AttributeEntityReference]] = provider.findAllReferences(entity)
 
     val expected = Map(
-      AttributeName.withDefaultNS("ref") -> Seq(AttributeEntityReference("refTypeA", "refName1")),
-      AttributeName.withDefaultNS("reflist") ->
+      AttributeEntityReference("name", "type") ->
```

Comment:
Should this be `AttributeEntityReference("type", "name")`?

Reply:
good catch, I had this backwards both here and at line 614 above. I am annoyed that `Entity` and `AttributeEntityReference` have their name and type arguments reversed … but I'm not about to change that